src/sentry/integrations/slack/integration.py

```diff
@@ -103,32 +122,28 @@ def get_team_info(self, access_token):
         resp.raise_for_status()
         status_code = resp.status_code
         resp = resp.json()
+        # TODO: track_response_code won't hit if we have an error status code
         track_response_code(status_code, resp.get("ok"))
 
-        return resp["team"]
-
-    def get_identity(self, user_token):
-        payload = {"token": user_token}
+        # TODO: check for resp["ok"]
 
-        session = http.build_session()
-        resp = session.get("https://slack.com/api/auth.test", params=payload)
-        resp.raise_for_status()
-        status_code = resp.status_code
-        resp = resp.json()
-        track_response_code(status_code, resp.get("ok"))
-
-        return resp["user_id"]
+        return resp["team"]
 
     def build_integration(self, state):
         data = state["identity"]["data"]
         assert data["ok"]
 
-        if settings.SLACK_INTEGRATION_USE_WST:
-            access_token = data["access_token"]
+        access_token = data["access_token"]
+        # bot apps have a different response format
+        # see: https://api.slack.com/authentication/quickstart#unfurling
+        if self.use_wst_app:
             user_id_slack = data["authorizing_user_id"]
+            team_name = data["team_name"]
+            team_id = data["team_id"]
         else:
-            access_token = data["bot"]["bot_access_token"]
-            user_id_slack = self.get_identity(data["access_token"])
+            user_id_slack = data["authed_user"]["id"]
+            team_name = data["team"]["name"]
+            team_id = data["team"]["id"]
```

Comment:
What is a "team" in slack? Is that a new term?

Reply:
@EvanPurkhiser They changed the format of the response. See: https://api.slack.com/authentication/quickstart#installing